model/TestCenterService.php

```diff
@@ -54,6 +64,14 @@ class TestCenterService extends tao_models_classes_ClassService
      */
     const ROLE_TESTCENTER_ADMINISTRATOR = 'http://www.tao.lu/Ontologies/TAOProctor.rdf#TestCenterAdministratorRole';
 
+    /**
+     * @deprecated
+     */
+    public static function singleton()
```

Comment:
What is the point of adding this deprecated method?

Reply:
This is needed for backward compatibility. There are a lot of places where `TestCenterService::singleton()` is called.